src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String SPECIFIED_CHARACTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        String[] result = new String[0];
+
+        if (isFileEmpty(file) == false) {
```

Comment:
```suggestion
        if (file.length() == 0) {
                return result;
        }
```
private method is redundant here)